### ISSUE TITLE
Error in ConfigurationCheck due to misspelling ($GLOBALS['TypoScriptFE'])

### ISSUE THREAD

**Issue by chrissonntag** [oliverklee/ext-oelib#876]
Installed version: 3.6.3
Class: OliverKlee\Oelib\Configuration\ConfigurationCheck
Lines: 1686-1689
Error: `$GLOBALS['TypoScriptFE']`does not exist and must be `$GLOBALS['TSFE']` instead
Leads to: `PHP Warning: Creating default object from empty value in /app/public/typo3conf/ext/oelib/Classes/Configuration/ConfigurationCheck.php line 436`
Code:
```
/**
 * @return TypoScriptFrontendController|null
 */
protected function getFrontEndController()
{
    return $GLOBALS['TypoScriptFE'] ?? null;
}

**Comment by oliverklee** [comment]
@chrissonntag Thanks for the report!

**Comment by oliverklee** [comment]
I've just released version 3.6.4 with the fix.
